requirements.txt

```diff
@@ -1,3 +1,8 @@
 # poetry not supported by cloud.gov, use this file instead for manually push
-# run this command to generate the file:
-# poetry export -f requirements.txt --output requirements.txt --without-hashes
\ No newline at end of file
+
+flask-migrate==4.0.7 
+flask-sqlalchemy==3.1.1 
+flask==3.0.2 
+psycopg2-binary==2.9.9 
+pytest==7.4.4 
+python-dotenv==1.0.1 
```

Comment:
Is this file supposed to be auto generated for cf push purpose, no need to keep an offline copy?

Reply:
you are right, just removed it.